frequently-asked-questions.md

```diff
@@ -0,0 +1,94 @@
+
+**Q: I see a lot of projects have been archived. Which projects are active right now?**
+A: We have several active projects going on and need work: Mentorship System (Android, Backend, Flutter, iOS),  anitab-org.github.io, VMS, Portal. If you want to work on an archived project, you can fork it and work on it independently. Your contributions will still be recognised. 
+
+**Q: I picked out a project. What do I do next?**
+A: Each project comes with a README which guides you on how to setup the project. If you face any problems you need to head over to the project’s Zulip channel and ask your question there. Someone from the community will help you out. If you need help right away, please mention @project-managers and someone will help you out. Once the project is setup, read the contributing guidelines to make your first contribution.
+
+## Open Source Programs
+
+**Q: Can previous ambassadors, apply to be an ambassador again?**
+A: Yes! You can re-apply.
+
+**Q: Do GCI/GSoC mentors receive compensation?**
+A: Please email gci-support@google.com or gsoc-support@google.com for inquiries.
+
+**Q: Can I contribute to GSoC projects even if I’m not a GSoC student?**
+A: Yes, GSoC projects are discussed in public channels because we want community involvement. You can even attend weekly meetings of each project if you want to! These projects are just as open source as they were before the GSoC selections. You can contribute with ideas, and even issues if they're not labeled as GSoC issues.
+
+**Q: Do I have to be a part of GCI/GSoC/Ambassadors/Maintainers to contribute to AnitaB.org Open Source?**
+A: No, you don’t need to be a part of those programs to contribute to AnitaB.org Open Source. Anyone can contribute to the open source community anytime.
+
+**Q: Do I have to be a part of Open Source Ambassadors to contribute to AnitaB.org Open Source?**
+A: No, you do not have to be a part of open source teams to contribute to AnitaB.org Open Source.
+
+## Systers to AnitaB.org Open Source change
+
+**Q: What is the difference between Systers Open Source and AnitaB.org Open Source?**
+A: We rebranded from Systers Open Source to AnitaB.org Open Source in 2020. GitHub projects that were active or still needed development have been moved to [github.com/anitab-org](http://github.com/anitab-org). 
+
+**Q: Are we still using Systers-open source on Slack?**
+A: Nope, we moved to Zulip - [AnitaB-org.zulipchat.com](https://anitab-org.zulipchat.com/) and anyone who was in our Slack was invited to the new workspace. We are keeping track of Android Tasks here and Backend Tasks here that were created during Google Code-in 2019.
```

Comment:
I think we have gone past the phase of keeping track of GCI tasks , all PRs and issues created during GCI are closed now on Github. The decision was taken during a PM session and May had aaproved closing all.
So the last sentence can be removed.

Reply:
great suggestion!